app/upgrades/v10/upgrades_test.go

```diff
@@ -0,0 +1,72 @@
+package v10_test
```

Comment:
nit, there are no tests beyond setting up the test suite. We can remove this here